docs/source/dev-guide/writing-tests/debugging-with-vscode.md

```diff
@@ -0,0 +1,30 @@
+Debugging Conda's Unit Test with VSCode
+=======================================
+
+This is one way to debug conda's unit tests using
+[VSCode](https://code.visualstudio.com/) on OSX.
+
+1. In a conda checkout, run `. dev/start -p 3.10` to enter a Python 3.10 (or the
+   version you want to test) development environment.
+2. Open VSCode from this terminal `code`.
+    > If `code` is not available, press `⌘-P`, `Shell` to find the `Install
+    > 'code' command in PATH` action.
+3. Ensure you have installed the `Python` extension.
```

Comment:
It might be helpful to even include a link to the Python extension being referred to. I'm assuming it's this one? https://marketplace.visualstudio.com/items?itemName=ms-python.python